docs/versioned_docs/version-v1.3.0/01-index.md

```diff
@@ -0,0 +1,45 @@
+---
+title: "Arcus - Messaging"
+layout: default
+slug: /
+sidebar_label: Welcome
+---
+
+# Introduction
+
+Arcus Messaging is a library that helps with the integration of messaging systems like queues or event subscriptions, with application message handling that describes the business.
+The library makes sure that the developer only has to concern themselves with the actual processing of messages instead of spending time with message peaking, connections, deserialization, and other infrastructure code that takes up time.
+
+Currently, we only support Azure Service Bus messaging but there's more to come!
```

Comment:
We also support EventHubs now in 1.3, don't we ?

Reply:
Ah yes, we can remove this line now. Good catch!